idiss/src/cs_exports.rs

```diff
@@ -355,6 +364,7 @@ pub unsafe extern "C" fn validate_recovery_request_cs(
         Err(e) => {
             let mut bytes = format!("{}", e).into_bytes();
             *out_length = bytes.len() as i32;
+            *out_capacity = bytes.capacity() as i32;
             let ptr = bytes.as_mut_ptr();
             std::mem::forget(bytes);
             ptr
```

Comment:
While this works, it would be better to use the newer ManuallyDrop since it is more clear
```
let wrapper = ManuallyDrop::new(bytes);
wrapper.as_ptr()
```